ztp/gitops-subscriptions/argocd/SNOExpansion.md

```diff
@@ -0,0 +1,483 @@
+      policyName: "config-policy"
+      metadata:
+        labels:
+          machineconfiguration.openshift.io/role: worker
+        name: enable-workload-partitioning
+      spec:
+        config:
+          storage:
+            files:
+            - contents:
+                source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZS53b3JrbG9hZHMubWFuYWdlbWVudF0KYWN0aXZhdGlvbl9hbm5vdGF0aW9uID0gInRhcmdldC53b3JrbG9hZC5vcGVuc2hpZnQuaW8vbWFuYWdlbWVudCIKYW5ub3RhdGlvbl9wcmVmaXggPSAicmVzb3VyY2VzLndvcmtsb2FkLm9wZW5zaGlmdC5pbyIKcmVzb3VyY2VzID0geyAiY3B1c2hhcmVzIiA9IDAsICJjcHVzZXQiID0gIjAtMyIgfQo=
+              mode: 420
+              overwrite: true
+              path: /etc/crio/crio.conf.d/01-workload-partitioning
+              user:
+                name: root
+            - contents:
+                source: data:text/plain;charset=utf-8;base64,ewogICJtYW5hZ2VtZW50IjogewogICAgImNwdXNldCI6ICIwLTMiCiAgfQp9Cg==
+              mode: 420
+              overwrite: true
+              path: /etc/kubernetes/openshift-workload-pinning
+              user:
+                name: root
+    - fileName: PerformanceProfile.yaml
+      policyName: "config-policy"
+      metadata:
+        name: openshift-worker-node-performance-profile
+      spec:
+        cpu:
+          # These must be tailored for the specific hardware platform
+          isolated: "4-47"
+          reserved: "0-3"
+        hugepages:
+          defaultHugepagesSize: 1G
+          pages:
+            - size: 1G
+              count: 32
+    - fileName: TunedPerformancePatch.yaml
+      policyName: "config-policy"
+      spec:
+        profile:
+          - name: performance-patch-worker
+            # The cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
+            data: |
+              [main]
+              summary=Configuration changes profile inherited from performance created tuned
+              include=openshift-node-performance-openshift-node-performance-profile
+              [bootloader]
+              cmdline_crash=nohz_full=4-47
+              [sysctl]
+              kernel.timer_migration=1
+              [scheduler]
+              group.ice-ptp=0:f:10:*:ice-ptp.*
+              [service]
+              service.stalld=start,enable
+              service.chronyd=stop,disable
+```
+##### __Comment 1__
+Set MCP to "worker"
+##### __Comment 2__
+This PTP configuration is added to demonstrate that in case ptp configuration on the worker is different from the master, a separate `PtpConfig` object can be added to match the worker configuration. In this example the additional `PtpConfig` is matched against a different label selector.
+##### __Comment 3__
+Please note that SR-IOV related configurations have a different policy name than the other objects in this file. It's because they have a different ZTP wave. and must be applied separately.
+This example also demonstrates a different node selector label.
```

Comment:
Is this required or optional?

Reply:
Same problem as above - we have already deployed everything with "master" selector, so now we have to use different selectors